src/main/resources/application.yml

```diff
@@ -0,0 +1,51 @@
+info.app:
+  name: HMPPS Offender Events UI
+  version: 1.0
+
+spring:
+  application:
+    name: offender-events-ui
+
+server:
+  port: 8095
+  servlet:
+    context-path: /
+  forward-headers-strategy: native
+  tomcat:
+    remote-ip-header: x-forwarded-for
+    protocol_header: x-forwarded-proto
+
+logging:
+  level:
+    org.springframework: INFO
```

Comment:
I don't like the logging levels here - duplication of logback spring

Reply:
Will drop in next PR